blueocean-dashboard/src/main/js/components/Pipelines.jsx

```diff
@@ -56,7 +56,11 @@ export default class Pipelines extends Component {
                 </PageHeader>
                 <main>
                     <article>
-                        <Extensions.Renderer extensionPoint="jenkins.pipeline.list.top" />
+                        { /* TODO: need to adjust Extensions to make store available */ }
```

Comment:
@kzantow I'm a little unclear on your comment. Redux `connect` will fail when used inside of an Extensions.Renderer because it appears it's being disconnected from the parent context, and connect requires the store be available either in the context or as a prop (which is the workaround I used). Ideally it seems like we shouldn't have to write any code to actually wire up the store?